keyboards/keyhive/maypad/keymaps/via/config.h

```diff
@@ -0,0 +1,19 @@
+/* 
```

Comment:
I mean the entire file, not just this line.

Reply:
Gotcha, I've removed the config.h file from keymaps/via